ucsmsdk/ucssession.py

```diff
@@ -463,13 +510,7 @@ def _login(self, auto_refresh=False, force=False):
         Returns:
             True on successful connect
         """
-
-        from .mometa.top.TopSystem import TopSystem
-        from .mometa.firmware.FirmwareRunning import FirmwareRunning, \
-            FirmwareRunningConsts
-        from .ucscoremeta import UcsVersion
         from .ucsmethodfactory import aaa_login
-        from .ucsmethodfactory import config_resolve_dn
 
         self.__force = force
 
```

Comment:
@vvb - `auto_refresh` should also be an attribute of `UcsSession`. 
`self.__auto_refresh = auto_refresh`.
This will help us in saving auto_refresh parameter, which will help us when serializing the handle.